src/runtime_src/core/pcie/linux/pcidev.h

```diff
@@ -64,9 +66,14 @@ struct fdt_header {
     uint32_t size_dt_struct;
 };
 
+// Forward declaration
+namespace pcidrv {
+class pci_driver;
+};
```

Comment:
Not sure if adding OS name in namespace is a good idea... it makes sense to have xrt_core::pci::drv and xrt_core::pci::dev. They are at the lowest level of SW stack closer to KMD. xrt_core::device_pcie can stay the same since it is upper level of device model providing ishim APIs. What do you think?

Reply:
xrt_core::pci::drv/dev works for me.

Reply:
@keryell That we won't do, `namespace xrt` is for publically exported XRT